### PR TITLE
feat: add finnish translations

### PR DESCRIPTION
**Pull request by rnousia**
## Description 

Added Finnish translations.

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Other

- [x] Non-breaking change
- [ ] Breaking change

## Developer checklist 

- [x] A [CHANGELOG entry] has been included (only when change is visible to users)

[CHANGELOG entry]: https://github.com/nlsfi/segment-reshape-qgis-plugin/blob/main/CHANGELOG.md

### CODE REVIEW

**Review by Joonalai — APPROVED** [review]
LGTM!

File: .gitlint [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+contrib=contrib-title-conventional-commits
+
+[title-max-length]
+line-length=72
+
+[title-min-length]
+min-length=10
+
+[title-match-regex]
+regex=: [^A-Z]
```
Comment (Joonalai) [inline .gitlint:16]: Does this mean that the title should start with capital letter?

Comment (rnousia) [reply .gitlint:16]: No, there is actually a colon at the start of the regex so it only checks that after a colon (and a space) there isn't a capital letter. I copied it from our other repositories' config although in this case we could just check that everything is in lowercase as we don't use ticketing system here. But I think that would be a bit too strict.

Comment (Joonalai) [reply .gitlint:16]: Oh right! The regex is a bit hard to read since it is not surrounded with quotes :)

**Pull request merged**